web/web/api/v1/notification.py

```diff
@@ -36,23 +36,23 @@ def get_notifications():
 
     notification_schema = NotificationSchema(
         only=fields_to_filter_on,
-        exclude=['created_by', 'alert_type_id', 'created_at', 'updated_at'])
+        exclude=['created_by', 'created_at', 'updated_at'])
 
     results = notification_schema.dump(notifications, many=True)
 
     return arw.to_json(results)
 
 
-@notifications_api_bp.route('/notification', methods=['PUT'])
+@notifications_api_bp.route('notification',
```

Comment:
why did you remove the `/`? Does it actually work w/o it?

Reply:
it works both ways, changed it to make it more consistent with the rest of them